client/assets/style/style.css

```diff
@@ -58,44 +86,51 @@ html, body {
 .text-view {
   min-height: 50vh;
   background-color: white;
-  margin: 20px;
-  padding: 15px;
-  border: 2px solid black;
-  border-radius: 25px;
+  margin: 20px 300px 20px 300px;
```

Comment:
if top & bottom, right & left are the same, you should consolidate the two.  margin: 20px 300px
http://www.w3schools.com/css/css_margin.asp for more info